modules/Microsoft.ContainerService/managedClusters/.test/flux/deploy.test.bicep

```diff
@@ -0,0 +1,105 @@
+targetScope = 'subscription'
```

Comment:
In general, is PR #2679 replacing this one? Should this be closed?

Reply:
I think that is a decision we need to make. What approach makes more sense of the two. The thrid approach isnt yet here which would be add extension into the fluxConfiguration module. Not sure that makes more or less sense though.

Reply:
Done in separate PR. Closing this as its not the option we want to go with.